srcpkgs/AusweisApp2/template

```diff
@@ -11,5 +11,5 @@ short_desc="Official authentication app for German ID cards and residence permit
 maintainer="Justin Jagieniak <justin@jagieniak.net>"
 license="EUPL-1.2"
 homepage="https://www.ausweisapp.bund.de/ausweisapp2/"
-distfiles="https://github.com/Governikus/AusweisApp2/archive/refs/tags/${version}.tar.gz"
-checksum=8bae7d4fcb03d1ab247bc7b35e55c5d4dfd98f017c121947652e16c29ae6049d
+distfiles="https://github.com/Governikus/${pkgname}/releases/download/${version}/${pkgname}-${version}.tar.gz"
```

Comment:
Please don't use pkgname in the URL.

Any reason you're switching to what looks like a release tarball?

Reply:
because I can't find the new version in tags...

Reply:
Ah sorry my fault ... I will fix it!